airflow/jobs.py

```diff
@@ -1434,7 +1434,7 @@ def _execute_helper(self, processor_manager):
                                                           State.NONE)
 
                 self._execute_task_instances(simple_dag_bag,
-                                             (State.SCHEDULED,))
```

Comment:
I know it's a workaround but since you mentioned it 
>unless you restart the scheduler

There are a couple of different options for doing micro restarts on the scheduler in 1.8: `run_duration` and `num_runs`. I think there are other benefits to restarting the scheduler this way, but I'm not sure on the details.